docs/user/eventdb.rst

```diff
@@ -10,6 +10,21 @@ The EventDB is not a software itself.
 
 The EventDB is a database (usually PostgreSQL) that gets filled with with data from intelmq using the SQL Output bot.
 
+-----------------------
+The events table itself
+-----------------------
+
+IntelMQ comes with the ``intelmq_psql_initdb`` command line tool. It creates an SQL file containing:
+
+- A ``CREATE TABLE events`` statement with all valid IntelMQ fields as columns and correct types
+- Several indexes as examples for a good read & search performance
+
+All elements of this SQL file can be adapted and extended by the database administrator to your needs, especially the indexes.
```

Comment:
I'd replace `by the database administrator to your needs` with: `before running the SQL file against a database`